infra/feast-operator/Makefile

```diff
@@ -99,6 +99,8 @@ help: ## Display this help.
 .PHONY: manifests
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
 	$(CONTROLLER_GEN) rbac:roleName=manager-role crd:maxDescLen=120 webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+	# Generate dynamic Helm chart from updated manifests
+	$(MAKE) helm-generate
```

Comment:
does it make more sense to run this here?
https://github.com/feast-dev/feast/blob/acb97fb4f92f014d5229180c51b4473855546219/infra/feast-operator/Makefile#L190-L193

Reply:
it's worth noting that `make build-installer` is executed with CI for each release, it also similarly already runs `kustomize build`